### PR TITLE
[APPC-4001] Fix hidden get button on small devices

### PR DESCRIPTION
**Pull request by rodolfobcardoso**
**What does this PR do?**
Fixes the get button on promotions for small devices.

**Database changed?**
No

**What are the relevant tickets?**
https://aptoide.atlassian.net/browse/APPC-4001

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] Database migration (if applicable)?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Functional QA tests pass

### CODE REVIEW

File: ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/ActivePromoCodeComposable.kt [hunk 1]
```diff
@@ -119,7 +127,13 @@ fun ActivePromoCodeComposable(cardItem: ActiveCardPromoCodeItem, fragmentName: S
                 fontSize = 14.sp
               )
             }
-            GetText(cardItem.action, cardItem.packageName, fragmentName = fragmentName, buttonsAnalytics = buttonsAnalytics)
+            GetText(
+              Modifier,
+              cardItem.action,
```
Comment (ricardo-frango) [inline ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/ActivePromoCodeComposable.kt:132]: add params names

File: ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/ActivePromoCodeComposable.kt [hunk 2]
```diff
@@ -119,7 +127,13 @@ fun ActivePromoCodeComposable(cardItem: ActiveCardPromoCodeItem, fragmentName: S
                 fontSize = 14.sp
               )
             }
-            GetText(cardItem.action, cardItem.packageName, fragmentName = fragmentName, buttonsAnalytics = buttonsAnalytics)
+            GetText(
+              Modifier,
```
Comment (ricardo-frango) [inline ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/ActivePromoCodeComposable.kt:131]: this param can be removed

File: ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt [hunk 3]
```diff
@@ -146,24 +167,18 @@ fun PromotionsCardComposable(cardItem: CardPromotionItem, fragmentName: String,
...
-                  cardItem.packageName,
-                  cardItem.hasVipPromotion,
-                  fragmentName,
-                  buttonsAnalytics
-                )
-              }
+              Spacer(modifier = Modifier.weight(1f, fill = true))
+              GetText(
+                modifier = Modifier.wrapContentWidth(),
+                cardItem.action,
```
Comment (ricardo-frango) [inline ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt:176]: add names to arguments

File: ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt [hunk 4]
```diff
@@ -175,7 +190,14 @@ fun PromotionsCardComposable(cardItem: CardPromotionItem, fragmentName: String,
               verticalAlignment = Alignment.CenterVertically
             ) {
               IconWithText(stringResource(id = R.string.perks_available_soon_short))
-              GetText(cardItem.action, cardItem.packageName, cardItem.hasVipPromotion, fragmentName, buttonsAnalytics)
+              GetText(
+                Modifier,
```
Comment (ricardo-frango) [inline ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt:194]: this param can be removed

File: ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt [hunk 5]
```diff
@@ -175,7 +190,14 @@ fun PromotionsCardComposable(cardItem: CardPromotionItem, fragmentName: String,
...
+                cardItem.action,
```
Comment (ricardo-frango) [inline ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt:195]: add names to params

File: ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt [hunk 6]
```diff
@@ -290,17 +312,26 @@ fun IconWithText(text: String) {
...
 
-  TextButton(onClick =  {
-    buttonsAnalytics?.sendDefaultButtonClickAnalytics(fragmentName, text)
-    action.invoke()
-  }) {
+  TextButton(
+    modifier = modifier,
+    onClick = {
+      buttonsAnalytics?.sendDefaultButtonClickAnalytics(fragmentName, text)
+      action.invoke()
```
Comment (ricardo-frango) [inline ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/PromotionsCardComposable.kt:333]: remove the invoke. leave just "action()"

**Review by ricardo-frango — CHANGES_REQUESTED** [review]

**Review by ricardo-frango — APPROVED** [review]

**Review by gustavoaptoide — APPROVED** [review]

**Pull request merged**